desktop/assets/auth.coffee

```diff
@@ -0,0 +1,9 @@
+require('backbone').$ = $
+
+routes =
+  '/signup': require('../apps/auth/client/auth.coffee').init
+
+  '/login': require('../apps/auth/client/auth.coffee').init
+
+for path, init of routes
+  $(init) if location.pathname.match path
```

Comment:
Just curious, when would the auth client not initialize? Is it more simple just to `init` always?